.github/workflows/vib-verify.yaml

```diff
@@ -0,0 +1,81 @@
+name: 'Early feedback'
+on: # rebuild any PRs and main branch changes
+  pull_request_target:
+    branches:
+      - master
+      - bitnami:master
+    types:
+      - assigned
+      - opened
+      - synchronize
+      - reopened
+    paths:
+      - 'bitnami/**'
+env:
+  CSP_API_URL: https://console.cloud.vmware.com
+  CSP_API_TOKEN: ${{ secrets.CSP_API_TOKEN }}
+  VIB_PUBLIC_URL: https://cp.bromelia.vmware.com
+jobs:
+  get-chart:
+    runs-on: ubuntu-latest
+    name: 'Get affected chart'
+    outputs:
+      chart: ${{ steps.get-chart.outputs.chart }}
+      result: ${{ steps.get-chart.outputs.result }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{github.event.pull_request.head.ref}}
+          repository: ${{github.event.pull_request.head.repo.full_name}}
+          fetch-depth: 0
+      - id: get-chart
+        run: |
+          files_changed="$(git diff --name-only origin/master HEAD)"
+          # Adding || true to avoid "Process exited with code 1" errors
+          charts_dirs_changed="$(echo "$files_changed" | xargs dirname | grep "bitnami/"| sort | uniq || true)"
+          # Using grep -c as a better alternative to wc -l when dealing with empty strings."
+          num_charts_changed="$(echo "$charts_dirs_changed" | grep -c "bitnami" || true)"
+          num_version_bumps="$(echo "$charts_dirs_changed" | grep Chart.yaml | xargs git diff origin/master | grep -c "+version" || true)"
```

Comment:
We're using a GH action called "trilom/file-changes-action" on other actions for this:

- https://github.com/bitnami/charts/blob/master/.github/workflows/generate-chart-readme.yml#L41

Feel free to keep it as it's.

Reply:
I'd keep it as it is as we need very specific conditions